src/mondeath.cpp

```diff
@@ -419,6 +419,7 @@ void mdeath::guilt( monster &z )
 
     Character &player_character = get_player_character();
     if( player_character.has_trait( trait_PSYCHOPATH ) || player_character.has_trait_flag( "PRED3" ) ||
+        player_character.has_trait_flag( "PACK_HUNTER" ) ||
```

Comment:
Again, if this does the same thing as PRED3, use a common flag. PRED4 (and all these traits) should use one generic flag for this check.